src/styles/panel.scss

```diff
@@ -18,6 +18,9 @@ $block: #{$fd-namespace}-panel;
     @include fd-reset();
   }
 
+  background-color: var(--fdPanel_Background);
+  border-radius: var(--fdPanel_Border_Radius);
+
   &__header {
```

Comment:
Should we have rounded `border-bottom` for collapsed panels?

<img width="933" alt="Screenshot 2021-11-04 at 12 55 01" src="https://user-images.githubusercontent.com/20265336/140302089-be9896e8-b9c9-48c5-b3a5-56352bd5bfce.png">